src/react-components/src/components/Link/Link.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from "react";
+import bem from "../../utils/bem";
+import { uid } from "react-uid";
+import { LinkTypes } from "./LinkTypes";
+import { element } from "prop-types";
+import Icon from "../Icons/Icon";
+import { iconRotationTypes } from "../Icons/IconTypes";
+
+export interface LinkProps {
+    /** Controls the link visuals—action, button, or default. */
+    linkType?: LinkTypes;
+
+    /** Href attribute */
+    href?: string;
+
+    /** BEM specifiers */
+    modifiers?: string[];
+
+    /** BEM specifiers */
+    blockName?: string;
+
+    /** Additional attributes, such as rel=nofollow, to pass to the <a> tag */
+    attributes?: {};
+}
+
+export default function Link(props: React.PropsWithChildren<LinkProps>) {
+    const {
+        href,
+        modifiers,
+        blockName,
+        attributes,
+        linkType = LinkTypes.Default,
+    } = props;
+
+    const passedProps = { href, attributes };
```

Comment:
@hokei There is probably a better way to do this, but I was getting an error in the console for passing extra stuff like "props.linkType" into the DOM element, so this was how I got around it 😬

Reply:
This is probably because the word `props` is reserved.  If you change `props` to `linkProps` the warning goes away and you can delete L34-37

Reply:
TIL!